jaxrs-api/src/main/java/jakarta/ws/rs/core/Response.java

```diff
@@ -67,6 +67,11 @@ protected Response() {
 
     /**
      * Get the complete status information associated with the response.
+     * <p>
+     * Comparing {@link StatusType} directly might unexpectedly return {@code false}.
```

Comment:
```suggestion
     * Comparing {@link StatusType} directly might unexpectedly return {@code false} if only the reason phrase differs.
```

The note should also explain why this can happen.